### ISSUE TITLE
Warning not announced to screenreader

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#258590]
Testing https://github.com/microsoft/vscode/issues/258230

With NVDA, I was able to tab to the shell integration link, but I was not able to see any announcement for the "Enable shell integration" message.

**Comment by Tyriar** [comment]
I wouldn't expect it to be announce it to screen readers via an alert but when they review the output however that happens. I asked a question and see this:

<img width="485" height="495" alt="Image" src="https://github.com/user-attachments/assets/fde0c100-c1df-4f24-a70c-f47966e7e402" />

It appears we don't show streaming output at all, when focusing the chat turn it also doesn't seem to read anything but the regular output parts:

<img width="1448" height="1003" alt="Image" src="https://github.com/user-attachments/assets/0e94b30b-f404-4035-a3e1-d42b5a8def0d" />

Assigning to @meganrogge as this seems to be a general problem parts, maybe just the terminal and with `ChatTerminalMarkdownProgressPart` and `ChatElicitationRequestPart` though?

**Comment by meganrogge** [comment]
We should be alerting as those come in and allowing the message to be focused via tab with an appropriate aria label

**Comment by meganrogge** [comment]
We now do alert the tool progress messages with `accessibility.verboseChatProgressUpdates` on by default. When that setting is enabled, we should also indicate this with an alert. 

It should also have a proper aria label. Right now, it's inaccessible.

**Comment by benvillalobos** [comment]
## To Test
- Launch NVDA screen reader
- Disable setting `Terminal > Integrated > Shell Integration: Enabled`
- Enter a simple prompt that triggers a terminal command into copilot chat (eg. `run ls command`)
- See that the notification "Enable shell integration to improve command detection" shows below the run of the command
- Listen for NVDA to mention "Enable shell integration <link> to improve command detection" as the notification appears
